pkgs/top-level/python-packages.nix

```diff
@@ -11868,6 +11868,12 @@ in {
     };
   };
 
+  wxPython_4_2 = callPackage ../development/python-modules/wxPython/4.2.nix {
+    wxGTK = pkgs.wxGTK31.override {
+      withGtk2 = false;
+      withWebKit = true;
+    };
+  };
```

Comment:
This is really not what we want for python Packages. How can we reduce the duplicated?